src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+        System.out.println("Hello I'm Duke\nWhat can I do for you?");
+        Scanner scan = new Scanner(System.in);
+        
+        while (true) {
+            String textInput = scan.nextLine();
+
+            if (textInput.equalsIgnoreCase("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                scan.close();
+                return;
+            }
+
+            if (textInput.equalsIgnoreCase("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskList.size(); i++) {
+                    System.out.println(i + 1 + ". " + taskList.get(i).toString());
```

Comment:
This manual string formatting works, however, have you considered using Java's builtin String.format method?